docs/data/date-pickers/lifecycle/lifecycle.md

```diff
@@ -20,10 +20,9 @@ The field components have an internal state controlling the visible value.
 
 It will only call the `onChange` callback when:
 
-- the user fills one section of an empty field. The value equals `Invalid date`.
 - the user completes all sections of a field. The value reflects the field.
-- the user cleans one section of a completed field. The value equals `Invalid date`.
-- the user cleans all sections of a field. The value equals `null`.
+  - the value could be `Invalid Date` in case of an unparsable date (i.e., if the [year is less than "100"](https://github.com/iamkun/dayjs/issues/1237) in case of `dayjs`).
```

Comment:
```suggestion
  - the value will be `Invalid Date` in case of an unparsable date (for example, if the [year is less than "100"](https://github.com/iamkun/dayjs/issues/1237) in case of `dayjs`).
```